packages/tradershub/src/screens/TermsOfUse/TermsOfUseSections/PEPs.tsx

```diff
@@ -10,7 +11,22 @@ const PEPs = () => (
             A politically exposed person (PEP) is someone appointed with a prominent public position. Close associates
             and family members of a PEP are also considered to be PEPs.
         </Text>
-        {/*  Add checkbox here once we have it in deriv-com/ui  */}
+        <Checkbox label='I am not a PEP and never have been a PEP.' />
```

Comment:
There are some issues in checkbox from ui. Once that is fixed will do the implementation 😸